### ISSUE TITLE
You should host a simple live demo of this

### ISSUE THREAD

**Issue by sean-stanley** [DevLab2425/angular-simple-slideshow#1]
A live demo to see it in action would be nice. I'm considering using in my project but without a demo I'd have to set it all up and see if it works the way I want and it might not have the features I need. I'll read the source code though and that may help but others would probably appreciate a codepen or something.

**Comment by DevLab2425** [comment]
Hey Sean,

Thanks for the note. I haven't maintained that repo in a while, but I'll
dig back in and see what I can do.
- Paul

## 

Paul Barry
Founder, DevLab2425

**Comment by sean-stanley** [comment]
Nice going. This repo doesn't have what I'm looking for out of the box this time but I may consider forking and adding some other features soon. Possibly convert to angular 1.5 component syntax while I'm at it.

Key things I'd want to add:
- add any html to a slide with ng-transclude
- pass back the current slide index in a callback to the parent controller (so I could add those little dots or something)
- slide transitions between slides with angular animate.

Great job at keeping this so light and simple by the way. Well done.

Basically I am making an on-boarding fullscreen slideshow for an app in a hurry.

**Comment by paul-barry-kenzan** [comment]
@sean-stanley 
Thanks for the input and I'd happily review any PR that you submitted. I haven't had the chance to upgrade much, but your suggestions seem like great enhancements.
